vars/longevityPipeline.groovy

```diff
@@ -20,92 +20,117 @@ def call(Map pipelineParams) {
             SCT_TEST_ID = UUID.randomUUID().toString()
             SCT_GCE_PROJECT = "${params.gce_project}"
         }
-        parameters {
+        options {
+            timestamps()
+            disableConcurrentBuilds()
+            buildDiscarder(logRotator(numToKeepStr: '20'))
+        }
+    stages {
+        stage('Parameters') {
```

Comment:
why did you change the structure compared what was before?

Reply:
my bad, I made changes for active-choices plugin. But thats not required for this PR, let me fix this with next update.